### PR TITLE
#534: selenium POC on browserstack

### PR DESCRIPTION
**Pull request by twschiller**
- [X] Run tests on BrowserStack as part of CI
- [x] Show ability to load Chrome extension
- [ ] Demonstrate ability to load Firefox extension
- [ ] Update to use a test runner (e.g., Jest)

### CODE REVIEW

File: selenium/sample-build.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+const webdriver = require("selenium-webdriver");
```
Comment (twschiller) [inline selenium/sample-build.js:18]: Codacy has a fix for the issue: [Do not use "require".](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7713940)
```suggestion
import webdriver from "selenium-webdriver";
```

Comment (twschiller) [reply selenium/sample-build.js:18]: Since the script is run via "node" and is not a module, the "import" is not supported

Comment (fregante) [reply selenium/sample-build.js:18]: It can be excluded via eslintrc as part of the `overrides`

Comment (twschiller) [reply selenium/sample-build.js:18]: I'll look at fixing the issues once we get a better idea of which test runner we want to use to control Selenium (e.g., Jest)

**Comment by twschiller** [comment]
Closing in favor of https://github.com/pixiebrix/pixiebrix-extension/pull/749

**Comment by fregante** [comment]
Testing something

**Review by fregante — DISMISSED** [review]
Testing

**Pull request closed without merging**